src/core/webidl.js

```diff
@@ -349,10 +349,10 @@ function renderWebIDL(idlElement, index) {
  */
 export function addIDLHeader(pre) {
   addHashId(pre, "webidl");
-  const header = hyperHTML`<div class="idlHeader"><a
+  const header = hyperHTML`<span class="idlHeader"><a
```

Comment:
Do we need this span? Can we merge this with the child anchor?

Reply:
We later add the webidl clipboard button in this wrapper